e2e/README.md

```diff
@@ -55,6 +55,17 @@ npm run cypress:run
 You can test against dev/acceptance environments as well. Just be
 sure to leave off the trailing `/`.
 
+### IAM Version
+
+Some tests will conditionally run in CI depending on what `CYPRESS_IAM_VERSION` is set to.
+To target those specific sets of tests set that environment variable when starting cypress:
+
+```bash
+CYPRESS_IAM_VERSION="v2.1" npm run cypress:run
```

Comment:
one more thing, I'd include the other possible values for IAM_VERSION
`export CYPRESS_IAM_VERSION= v1 | v2 | v2.1`

and maybe a more emphatic `Your dev environment's IAM version MUST match the value of CYPRESS_IAM_VERSION for the tests to pass locally`